shared/src/packages/drivers/libs/validation-schemas/driver-update-request-body.validation-schema.ts

```diff
@@ -0,0 +1,23 @@
+import joi from 'joi';
+
+import { commonSignUpRules } from '~/packages/users/libs/validation-schemas/common-rules/common-rules.js';
+
+import { DriverValidationMessage } from '../enums/enums.js';
+import { type DriverUpdateRequestDto } from '../types/types.js';
+
+const driverUpdateRequestBody = joi.object<
+  Omit<DriverUpdateRequestDto, 'businessId'>,
+  true
+>({
+  ...commonSignUpRules,
+  driverLicenseNumber: joi
+    .string()
+    .trim()
+    .regex(/^[A-Z]{3} \d{6}$/)
```

Comment:
Noticed that it's already defined in shared/src/packages/users/libs/validation-schemas/enums/user-validation-rule.enum.ts

Reply:
Yes, I added it there myself, but it completely slipped my mind. Thank you for noticing.